app/controllers/reviews_controller.rb

```diff
@@ -43,14 +43,24 @@ def destroy
     @superpower = @review.superpower
 
     @review.destroy
+
     flash[:notice] = 'Review deleted'
 
     redirect_to superpower_path(@superpower)
-
   end
 
   private
 
+  def authorize_user
```

Comment:
It doesn't look like we're actually using this method at all, please either implement or remove!

Reply:
will implement in a `before_action` call